docker/Dockerfile_simulation-bionic

```diff
@@ -33,10 +33,5 @@ RUN git clone --depth 1 https://github.com/mavlink/c_library_v2.git /usr/local/i
 # Some QT-Apps/Gazebo don't not show controls without this
 ENV QT_X11_NO_MITSHM 1
 
-# Gazebo 7 crashes on VM with OpenGL 3.3 support, so downgrade to OpenGL 2.1
```

Comment:
@TSC21 I would not remove this since this problem is still present in my up to date VMWare setup with Ubuntu 20.04 and gazebo 9. That's why the perfectly working workaround is in the setup script I created as well: https://github.com/PX4/Firmware/blob/bda072512a97eac74b3c74ce07cbc0dcf09db5d6/Tools/setup/arch.sh#L146-L149

We should make sure people and servers using VMs don't run into this. I'm not sure why it's in the container script but it could well be that if you don't start gazebo headless they'll crash because of this.

Reply:
Interesting. Let me have a look. We can update it further no problem.